webserver/views/user.py

```diff
@@ -98,7 +104,35 @@ def import_data():
                            lastfm_username=lastfm_username)
 
 
+@user_bp.route("/export")
+@login_required
+def export_data():
+    """ Exporting the data to various formats, like, json """
+    filetype = request.args.get("type")
+    if filetype == "json":
```

Comment:
I don't think this is a good way to determine if we need to return data or render the interface. How about sending POST request with format of the dump?

I'm also not sure if specifying the format is necessary. Are there any concrete plans to add alternatives in near future? If not, this seems useless.